.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   },
   rules: {
     'comma-dangle': ['error', 'always-multiline'],
+    'indent': ['error', 2, { SwitchCase: 0 }],
```

Comment:
Note that this PR assumes we want to have `case` statements at the same level as their containing `switch`, which seems like the idiom within Redux but deviates from our otherwise consistent nesting. I have no real preference so if @cowboy or @gnarf or @boazsender prefer nesting we can revert this ESLint rule change and indent all switch block bodies